### ISSUE TITLE
[Usage]: how to acquire logits in vllm

### ISSUE THREAD

**Issue by zeyang12-jpg** [vllm-project/vllm#8762]
### Your current environment

```text
The output of `python collect_env.py`
```

### How would you like to use vllm

I want to acquire logits when I run benchmark_throughput.py to do the softmax optimization, but the output in vllm doesn't have logits, how can I acquire it.

### Before submitting a new issue...

- [X] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by zeyang12-jpg** [comment]
my env is MI250 and ROCM 6.1

**Comment by hmellor** [comment]
Would a `logits_processor` work for your use case? https://docs.vllm.ai/en/latest/dev/sampling_params.html
